styleguide/styles.scss

```diff
@@ -220,3 +220,7 @@
   border: 1px solid $cads-language__border-colour;
   margin-bottom: 1rem;
 }
+
+.sb-show-main {
```

Comment:
Looks like the new storybook version adds extra padding to the body. Removed it to keep the look (and VRTs) consistent.